test/functionalTest/cases/0000_subs_attrs_deletion/delete_attribute_in_condition.test

```diff
@@ -207,7 +207,8 @@ Date: REGEX(.*)
                 "attrs": [
                     "B",
                     "C"
-                ]
+                ],
+                "notifyOnMetadataChange": false
```

Comment:
Let's clarify the semantics of this new `notifyOnMetadataChange` field.

* `"notifyOnMetadataChange": true`. The metadata is considered part of the value of the attribute in the context of notification, so if the value doesn't change but the metadata changes, then a notification is triggered.
* `"notifyOnMetadataChange": false`. The metadata is not considered part of the value of the attribute in the context of notification, so if the value doesn't change but the metadata changes, then a notification is *not* triggered.

If we agree on that semantic (otherwise, please do an alternative proposal), I understand that to keep backward compatibility we should use `true` here (and in other existing .test files).

Reply:
...assuming that the current behaviour (i.e. before this PR) is that the value doesn't change but the metadata changes, then a notification is triggered. It would be great if you could check that with the last ContextBroker version available (3.8.1)

Reply:
I agree with the semantics, but a subscription with an empty condition ("notifyOnMetadataChange") returns false because no notifyOnMetadataChange value was provided.

Reply:
In general, empty field (whatever) means "default values for the sub-field inside the field".

Thus, empty `condition` means "default values for the sub-fields inside `condition`, being `notificationOnMetadataChange` a sub-field of `condition` which default is `true` (as discussed above).

Reply:
understood, I'll change the test cases to reflect your recommendation and set the notificationOnMetadataChange default value to true.